tests/publiccloud/boottime.pm

```diff
@@ -271,6 +271,7 @@ sub do_systemd_analyze {
 
     # Wait for guest register, before calling syastemd-analyze
     $instance->wait_for_guestregister();
+    $instance->check_guestregister();
```

Comment:
I think this test should be exception . check one line below . following measurement does not make sense with line which you adding here ...